doc/server/hap2/how-to-setup-livestatus.md

```diff
@@ -0,0 +1,105 @@
+# How to setup Livestatus API
+-----------------------------------------------------------------------------------
+
+You should install livestatus module, if you use hap2_nagios_livestatus.py plugin.
+
+This document describe for how to install livestatus module on CentOS 7 and Ubuntu 14.04 LTS.
+
+This document is written with the assumption that Nagios is installed.
+
+## (1) Install the livestatus module package
+
+### CentOS 7
+
+    # yum install epel-release
+    # yum install check-mk-livestatus 
+
+### Ubuntu 14.04 LTS
+
+    # apt-get install check-mk-livestatus
+
+## (2) Check livestauts library path
+
+You execute the following command on each OS to check the path of "livestatus.o".
+
+The path is replaced the next section's LIVESTATUS_LIB_PATH.
```

Comment:
I cannot find 'LIVESTATUS_LIB_PATH' below.
You mean it's LIVESTATUS_LIB_FILE_PATH ?
If so,  I suggest the following sentence.

    The path obtained above should be LIVESTATUS_LIB_FILE_PATH in the next section.